types/oxon/oxobutton2/default.schema.json

```diff
@@ -0,0 +1,86 @@
+{
+  "$id": "https://akenza.io/oxon/oxobutton2/default.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "processingType": "uplink_decoder",
+  "topic": "default",
+  "title": "Default",
+  "properties": {
+    "button1": {
+      "title": "Button 1",
+      "type": "integer",
+      "description": "Button 1 got pressed if the number is 1",
+      "hideFromKpis": true
+    },
+    "button2": {
+      "title": "Button 2",
+      "type": "integer",
+      "description": "Button 2 got pressed if the number is 1",
+      "hideFromKpis": true
+    },
+    "button3": {
+      "title": "Button 3",
+      "type": "integer",
+      "description": "Button 3 got pressed if the number is 1",
+      "hideFromKpis": true
+    },
+    "button4": {
+      "title": "Button 4",
+      "type": "integer",
+      "description": "Button 4 got pressed if the number is 1",
+      "hideFromKpis": true
+    },
+    "periodicUplink": {
+      "title": "Periodic uplink",
+      "type": "integer",
+      "description": "Its a periodic uplink if the number is 1",
+      "hideFromKpis": true
+    },
+    "heartbeatEvent": {
+      "title": "Heartbeat event",
+      "type": "integer",
+      "description": "Its a heartbeat event if the number is 1",
+      "hideFromKpis": true
+    },
+    "temperatureEvent": {
+      "title": "Temperature event",
+      "type": "integer",
+      "description": "Its a temperature event if the number is 1",
+      "hideFromKpis": true
+    },
+    "humidityEvent": {
+      "title": "Humidity event",
+      "type": "integer",
+      "description": "Its a humidity event if the number is 1",
```

Comment:
use booleans for this? 
typo: It is

Reply:
I can, but integers make it easier for summating datapoints on the dashboard